bench: flake in rttanalysis/BenchmarkORMQueries/hasura_column_descriptions

Found here: https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Ci_Tests_Bench/7779209?showRootCauses=false&expandBuildChangesSection=true&expandBuildProblemsSection=true

```
[21:50:51] :			 [Bench //pkg/bench/rttanalysis:rttanalysis_test] --- FAIL: BenchmarkORMQueries/hasura_column_descriptions
[21:50:51] :			 [Bench //pkg/bench/rttanalysis:rttanalysis_test]     test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/1b686e68658fd1761e16df16f8888527/logBenchmarkORMQueries_hasura_column_descriptions2163865204
[21:50:51] :			 [Bench //pkg/bench/rttanalysis:rttanalysis_test]     test_log_scope.go:79: use -show-logs to present logs inline
[21:50:51] :			 [Bench //pkg/bench/rttanalysis:rttanalysis_test]     rtt_analysis_bench.go:179: BenchmarkORMQueries/hasura_column_descriptions: got 83, expected 85
[21:50:51] :			 [Bench //pkg/bench/rttanalysis:rttanalysis_test]     rtt_analysis_bench.go:185: wrote jaeger trace to /artifacts/tmp/_tmp/1b686e68658fd1761e16df16f8888527/logBenchmarkORMQueries_hasura_column_descriptions2163865204/BenchmarkORMQueries_hasura_column_descriptions.jaeger.json
[21:50:51] :			 [Bench //pkg/bench/rttanalysis:rttanalysis_test]     testing_shims.go:42: -- test log scope end --
[21:50:51] :			 [Bench //pkg/bench/rttanalysis:rttanalysis_test] test logs left over in: /artifacts/tmp/_tmp/1b686e68658fd1761e16df16f8888527/logBenchmarkORMQueries_hasura_column_descriptions2163865204
[21:52:33] :			 [Bench //pkg/bench/rttanalysis:rttanalysis_test] --- FAIL: BenchmarkORMQueries/hasura_column_descriptions_8_tables
[21:52:33] :			 [Bench //pkg/bench/rttanalysis:rttanalysis_test]     test_log_scope.go:161: test logs captured to: /artifacts/tmp/_tmp/1b686e68658fd1761e16df16f8888527/logBenchmarkORMQueries_hasura_column_descriptions_8_tables1484250699
[21:52:33] :			 [Bench //pkg/bench/rttanalysis:rttanalysis_test]     test_log_scope.go:79: use -show-logs to present logs inline
[21:52:33] :			 [Bench //pkg/bench/rttanalysis:rttanalysis_test]     rtt_analysis_bench.go:179: BenchmarkORMQueries/hasura_column_descriptions_8_tables: got 83, expected 85
[21:52:33] :			 [Bench //pkg/bench/rttanalysis:rttanalysis_test]     rtt_analysis_bench.go:185: wrote jaeger trace to /artifacts/tmp/_tmp/1b686e68658fd1761e16df16f8888527/logBenchmarkORMQueries_hasura_column_descriptions_8_tables1484250699/BenchmarkORMQueries_hasura_column_descriptions_8_tables.jaeger.json
[21:52:33] :			 [Bench //pkg/bench/rttanalysis:rttanalysis_test]     testing_shims.go:42: -- test log scope end --
[21:52:33] :			 [Bench //pkg/bench/rttanalysis:rttanalysis_test] test logs left over in: /artifacts/tmp/_tmp/1b686e68658fd1761e16df16f8888527/logBenchmarkORMQueries_hasura_column_descriptions_8_tables1484250699
[21:52:35] :			 [Bench //pkg/bench/rttanalysis:rttanalysis_test] BenchmarkORMQueries/hasura_column_descriptions_modified-16                          	       1	 790635693 ns/op	         6.000 roundtrips
```

emphasis:
```
BenchmarkORMQueries/hasura_column_descriptions: got 83, expected 85
```



Jira issue: CRDB-21960

--- a/pkg/bench/rttanalysis/orm_queries_bench_test.go
+++ b/pkg/bench/rttanalysis/orm_queries_bench_test.go
@@ -287,8 +287,9 @@ ORDER BY relname DESC, input`,
 		},
 
 		{
-			Name:  "hasura column descriptions",
-			Setup: "CREATE TABLE t(a INT PRIMARY KEY)",
+			Name:      "hasura column descriptions",
+			SkipIssue: 88885,
+			Setup:     "CREATE TABLE t(a INT PRIMARY KEY)",
 			Stmt: `WITH
   "tabletable" as ( SELECT "table".oid,
            "table".relkind,

